org.freedesktop.Sdk.Extension.openjdk.appdata.xml

```diff
@@ -15,4 +15,7 @@
   <url type="bugtracker">https://bugs.openjdk.java.net/</url>
   <developer_name>Mat Booth</developer_name>
   <update_contact>mat.booth@gmail.com</update_contact>
+  <releases>
+    <release version="19.0.2+7" date="2023-02-21"/>
```

Comment:
hrmpf.
I thought there was a way of "suggesting" an edit by performing it directly. But I can't find it now.
It currently says "19", but you're building "20", no?

Reply:
Thanks for pointing that out. I forgot to do a search and replace for 19.

Reply:
And fixed -- that was automatically added when I used `org.flathub.flatpak-external-data-checker` to run an update. I've dropped the `<releases>` node since there never was release with Java 19.